zipkin-common/src/main/scala/com/twitter/zipkin/test/FunSuiteWithJUnit.scala

```diff
@@ -0,0 +1,9 @@
+package com.twitter.zipkin.test
+
+import org.scalatest.junit.JUnitRunner
+import org.junit.runner.RunWith
+import org.scalatest.FunSuite
+
+@RunWith(classOf[JUnitRunner])
+class FunSuiteWithJUnit extends FunSuite {
```

Comment:
PS I would help with pulling this out.. just don't want anything else depending on common than absolutely must :P